source/get-started/patternfly-angular-migration-guide-3.0/index.md

```diff
@@ -0,0 +1,84 @@
+---
```

Comment:
Instead of having separate directories, each with their own index.md files, can we provide explicit file names? For example:

source/patternfly-migration-guides/angular-patternfly-migration-guide-3.0.md
source/patternfly-migration-guides/patternfly-migration-guide-2.0.md
source/patternfly-migration-guides/patternfly-migration-guide-3.0/index.md

Also note the "angular-patternfly" file name.